google-cloud-bom/pom.xml

```diff
@@ -258,6 +258,11 @@
         <artifactId>grpc-google-cloud-bigquerydatatransfer-v1</artifactId>
         <version>0.83.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-bigquerydatatransfer-v1:current} -->
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-bigquerystorage</artifactId>
+        <version>0.118.1-beta-SNAPSHOT</version><!-- {x-version-update:google-cloud-bigquerystorage:current} -->
```

Comment:
This shouldn't be a snapshot version and maybe not 0.118.1

Reply:
Done.